src/components/heading/heading.pw.tsx

```diff
@@ -0,0 +1,358 @@
+/* eslint-disable jest/no-conditional-expect, jest/valid-expect */
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import {
+  HeadingComponent,
+  HeadingComponentWithPills,
+} from "./components.test-pw";
+import {
+  headingPreview,
+  headingTitle,
+  headingHelp,
+  headingHelpTooltip,
+  subheaderPreview,
+  dividerPreview,
+  separatorPreview,
+  pillPreview,
+} from "../../../playwright/components/heading/index";
+import { link } from "../../../playwright/components/index";
+import { CHARACTERS } from "../../../cypress/support/component-helper/constants";
+import { HeadingType } from "../../../src/components/heading";
+import {
+  checkAccessibility,
+  getStyle,
+} from "../../../playwright/support/helper";
+
+const specialCharacters = [CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS];
+const testData = ["https://carbon.sage.com/"];
+const dividerAndSeparatorValue = [true, false];
+const headingType: HeadingType[] = ["h1", "h2", "h3", "h4", "h5"];
+
+test.describe("should render Heading component", () => {
```

Comment:
suggestion: I think these test descriptions could be a little more clearer so they act as better documentation etc

```suggestion
test.describe("Heading component", () => {
```